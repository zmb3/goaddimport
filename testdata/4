package main

import (
	"bytes"
	"fmt"
)

func main() {
	fmt.Println(&bytes.Buffer{})
}
