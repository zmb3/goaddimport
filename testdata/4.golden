package main

import (
	"bytes"
	"fmt"

	"github.com/someuser/somepkg"
)

func main() {
	fmt.Println(&bytes.Buffer{})
}
